sql/moz-fx-data-shared-prod/stripe/itemized_payout_reconciliation/view.sql

```diff
@@ -45,34 +64,96 @@ charge_states AS (
   WHERE
     cards.country IN ("US", "CA")
 ),
+subscriptions AS (
+  SELECT
+    subscription_id,
+    plan_id,
+    plan_name,
+    plan_interval,
+    plan_interval_count,
+    product_id,
+    product_name,
+  FROM
+    `moz-fx-data-shared-prod`.subscription_platform.stripe_subscriptions
+),
 enriched AS (
   SELECT
-    report.*,
-    charge_states.state,
-    charge_states.postal_code,
+    report.* EXCEPT (
+      card_country,
+      shipping_address_city,
+      shipping_address_country,
+      shipping_address_line1,
+      shipping_address_line2,
+      shipping_address_postal_code,
+      shipping_address_state
+    ),
+    {% for country, state, postal_code, dst_pre in [
```

Comment:
Looks like the SQL generation CI is failing on the Jinja syntax, so maybe Jinja is only currently supported in queries, not views?

Since you verified that no current customer shipping or billing addresses need these US territory overrides, I'm fine if you want to reverse course and only apply this logic to the card/charge addresses like before.

Reply:
I'm pretty sure jinja is supposed to be supported in views, considering I was able to deploy it to a dev environment, it's just that no one has actually used it in views yet so CI needs some touch ups.